monitor/api/src/main/java/org/datacleaner/monitor/configuration/TenantContext.java

```diff
@@ -162,4 +163,6 @@
      * @return
      */
     public ComponentStore getComponentStore();
+
+    public StringConverter getStringConverter();
```

Comment:
I also feel not comfortable with this part. Maybe you can propose another way how to make the StringConverter more performant? Especially the deserialization. I feel my lack of a global overview of DataCleaner parts.